src/types.d.ts

```diff
@@ -3843,7 +4322,7 @@ declare namespace bigquery {
       /**
        * Filter for job state
        */
-      stateFilter?: 'done' | 'pending' | 'running';
+      stateFilter?: Array<'done' | 'pending' | 'running'>;
```

Comment:
Might be another nice place to break out types, though again if it's generated, probably not much to do...